scripts/package/deb-after-install.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
```

Comment:
I'm personally not a big fan of those scripts as they assume that the global configuration is already in place for the post-install scripts to run. If you want a simple package that can be redistributed, you don't want to hardcode the postgresql host and credential in there for alembic to do the upgrades.
The package is also creating all the configuration files and overwriting any existing files, so it's not safe to assume the standard config files are enough for a proper install. A lot of configuration needs to be reapplied after a package install or upgrade.
I'd be more happy with a set of clean puppet files on the side to do everything as needed!